packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -17,8 +17,29 @@ import { Op } from "./opcode";
 
 const BIGINT0 = BigInt("0");
 const BIGINT1 = BigInt("1");
+
+// Store TEAL version
+export class Pragma extends Op {
+  readonly version;
+
+  constructor (arg: string[], line: number) {
+    super();
+    assertFieldLen(arg.length, 2, line);
+    if (arg[0] === "version") {
+      this.version = arg[1];
+    } else {
+      throw new TealError(ERRORS.TEAL.INVALID_OP_ARG, { line: line });
+    }
+  };
+}
+
 // pops string([]byte) from stack and pushes it's length to stack
 export class Len extends Op {
+  constructor (arg: string[], line: number) {
+    super();
+    assertFieldLen(arg.length, 0, line);
```

Comment:
How about renaming: `assertFieldLen` -> `assertLen`?

Reply:
yes, we renaming fields also, so `assertLen` is better